recipes/Kili_fine_tuning_tutorial.ipynb

```diff
@@ -0,0 +1,2081 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.        max_tokens=8,
</pre><p>you should explain how you determine the max number of tokens to generate</p><p> </p><p>by using the openai tokenizer for example: https://platform.openai.com/tokenizer</p><p> </p><p>the longest class has 7 tokens, with some margin (since the LLM can add spaces somtimes), it's best to fix it to 10 or 15</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/1259/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>Done</p>